README.md

```diff
@@ -175,10 +175,11 @@ Arrow2 also has a more granular code base, helping to reduce the compiler bloat.
 
 Development of Polars is proudly powered by
 
-[![Xomnia](https://raw.githubusercontent.com/pola-rs/polars-static/master/sponsors/xomnia.png)](https://www.xomnia.com/)
+
```

Comment:
My employer sponsors my time, so I'd want them to keep a prominent place. :)

Can we put the jetbrains logo under sponsors? Before the link or something like that?

Reply:
Done, and made the Xomnia link also a logo in sponsors. Now it appears twice, but otherwise it is odd that JB is a logo and Xomnia is not?

Reply:
:+1: Thanks! :)